regression/cbmc/r_w_ok_inconsistent_valid_negated/main.c

```diff
@@ -0,0 +1,54 @@
+#include <assert.h>
+#include <stdlib.h>
+
+char c1;
+
+char *p1;
+char *p2;
+char *p3;
+
+void test_global_pointer()
+{
+  char c2;
+
+  p1 = &c1;
+  p2 = &c2;
+
+  p3 = malloc(1);
+  __CPROVER_assume(p1 != NULL);
+  
+  __CPROVER_assume(
+    !__CPROVER_r_ok(p1, 1) ||
```

Comment:
I'd highlight the negation here with a comment explaining the purpose

Reply:
Ehh only if you think worth it - it is of course also explained in the description